app/jobs/cache/browser_channels/responses_for_prefix.rb

```diff
@@ -72,13 +80,15 @@ def get_paypal_wallet_state(paypal_connection:)
     end
   end
 
+  def get_bitflyer_wallet_state(bitflyer_connection:)
+    PublishersPb::BitflyerWalletState::BITFLYER_ACCOUNT_KYC
```

Comment:
Just to confirm, we're assuming that every BitFlyer who registers with us is KYC'd already, correct?

Reply:
Yes this is correct, confirmed this with Chris today